src/main/java/com/robin/hoteless/Controllers/GreetingController.java

```diff
@@ -0,0 +1,20 @@
+package com.robin.hoteless.Controllers;
+
+import com.robin.hoteless.App.Greeting;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+public class GreetingController {
+    private static final String template = "Hello, %s!";
```

Comment:
Why static? Also, constant/final String names should be ALL_CAPS_WITH_UNDERSCORES